manifests/overlays/prod/secrets/clusters/apps.ocp.prod.psi.redhat.com.enc.yaml

```diff
@@ -10,7 +10,7 @@ type: Opaque
 stringData:
     name: apps.ocp.prod.psi.redhat.com
     config: ENC[AES256_GCM,data:Br9euMohBhBWzKh7yfOuqZsbvBMGEhUcfDIKqTr+e6YsDefQ3di9BIR37QeNhmtx3tt00tq4UQJxn3CvCJbXbZOX7dvtPT6OW6gvI1TxT7bdmvw847MVjhZi8/J3ncloPBSvwOe2DZI65yJosrdC5rE/mrmaKOjZLVCqTiXln1Orl/0dHq7Q4JzYf/lTSiFq9T7JusW5ByxXSzmLXyNyTx86r5e2hPP3IiUsYveRaYOuicgRa7cq+abASgzJh1+B389wKUCemxrQyetFTRC74DXTmn9vQP7CcU+hFR37vqB86wL5ss3zo/45w+ntx758qd3SbI860oX8+6iizM/DNu9OThQeKx8D5Eb5g6r3kv14hZiIPSArLBh2Ult87a29/2o2ojf3rwK2ykaqSr9pDe2woA+/DtfWEa3YJStWU9mZpvm9Zvp9zEH5sV/CFedzlFqNK/9EUyzVxiEqWTNYmimg8qd9xx7jRL+1lpLYGFyo85GlEf4B2LvfYJmpNRUTRMHX0nAc1mLkhBbFiO3C6LH4jYcAdj9WZppSAHIIlrnwCTHuE6eydAyN0XQmM32CnY12isyW8Qx13U0sYWYmC/ySt1fEup/1FJH0Gp2RctNcwiL+ydfYQaa6GhmCO1FjOO65kJ4odZ1PVaR8h/JeSVbVAZP9K41Mvh9+1jYRrIVJzY/7l2KpKHRtALSyYMdjFdrSf6d8owO6pJ8+DFEw0iVJT5fBC02/sfxT+d+/Mp1gTUxCZko2S03Avqpk57gm2Q1TgKWFiSMWVn2WNie9J0ufZJ1NliTonrVXoWVl1+Pzbf7X6xEt7UFEiVKSte7xLIGaw0H8wDLShSgClweBc16AUWUZk1WuAr35t6dzp7YJDDuO9hDIvKZFg8gvM1fkU+CGzpDETZE8r4V0tCu7Uo65dwvZ6Sgn3HhM8prlc1pvW4OnSAy1FIgMcoScEbl0n8KySaHjLgWYKBI+H3H6hpCcM2ouc3QLWVIy9N2vjLoBHpq2PclhG2eJhv8Ljbs3qISWfZZ4p5ufiNKujP91VxhIKUvlmcCYb27x6db7tfo2Mrh9VrEVtqn3Y7b5SKp0gMhn0uqt33fkgxdNqYu/TafsEnkAPXco4p+yJ4Ks8h2eBq3KBmsEjetMMeoP+8AXNA0gvk28Hego2PbF+fZOhJfhDJat3ivMH2xmTOJudYXOLGw8NvJfMxpQlA1w3hiiDQGx5YAz+/pvJ5Aa/Yy+4ZLVpSmShVcf1LGhEqmD5NuCxmJF/jxqVTGjVm3gK0lZJ5nsqQODJSl84I3Hee0lxnqI705kIDMI/mb/byoCKoFRGNC880Whjw==,iv:pMTOKQYhOCSXVtcJL2Bk+2cS1MaUGaR66d/J3Mm8Vb8=,tag:QnJ/BIY1UIlbjEogTLkcnQ==,type:str]
-    namespaces: aicoe-argocd,dtuchyna-thoth-dev,fmurdaca-thoth-dev,fpokorny-thoth-dev,goern-thoth-dev,kpostlet-thoth-dev,thoth-amun-api-stage,thoth-amun-inspection-stage,thoth-backend-stage,thoth-frontend-stage,thoth-graph-stage,thoth-infra-stage,thoth-middletier-stage,thoth-test-core,aicoe,aicoe-prod-bots
+    namespaces: aicoe-argocd,dtuchyna-thoth-dev,fmurdaca-thoth-dev,fpokorny-thoth-dev,goern-thoth-dev,kpostlet-thoth-dev,thoth-amun-api-stage,thoth-amun-inspection-stage,thoth-backend-stage,thoth-frontend-stage,thoth-graph-stage,thoth-infra-stage,thoth-middletier-stage,thoth-test-core,aicoe,aicoe-prod-bots,aiops-prod-argo
```

Comment:
you have to sops edit this file, i.e. 

`$ sops apps.ocp.prod.psi.redhat.com.enc.yaml ` 

Then add the namespace in your editor and save/exit